app/views/layouts/application.html.erb

```diff
@@ -10,90 +10,195 @@
+              <% end %><br />
+              <%= f.password_field :password, autocomplete: "off", class: 'col-xs-12' %>
+            </div>
+            <br />
+            <br />
+            <div class="field">
+              <%= f.label :password_confirmation %><br />
+              <%= f.password_field :password_confirmation, autocomplete: "off", class: 'col-xs-12' %>
+            </div>
+            <br />
+            <br />
+
+          <%= render "devise/shared/links" %>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          <%= f.submit "Sign Up", class: 'btn btn-primary' %>
+        </div>
         <% end %>
       </div>
-      <div class="modal-footer">
-      </div>
     </div>
   </div>
-</div>
 
+    <!-- Modal for Log In-->
+    <div class="modal fade" id="myModalForLogIn" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+            <h2 class="modal-title text-center" id="myModalLabel">Log In</h4>
+          </div>
+          <br />
+          <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+          <div class="modal-body">
+              <div class="field">
+                <%= f.label :email %><br />
+                <%= f.email_field :email, autofocus: true, class: 'col-xs-12' %>
+              </div>
+              <br class="clr"/>
+              <br />
+              <div class="field">
+                <%= f.label :password %><br />
+                <%= f.password_field :password, autocomplete: "off", class: 'col-xs-12' %>
+              </div>
+              <br class="clr"/>
+              <br />
+              <% if devise_mapping.rememberable? -%>
+                <div class="field">
+                  <%= f.check_box :remember_me %>
+                  <%= f.label :remember_me %>
+                </div>
+              <% end -%>
+            <br class="clr"/>
+            <%= render "devise/shared/links" %>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+            <%= f.submit "Log In", class: 'btn btn-primary' %>
+          </div>
+          <% end %>
+        </div>
+      </div>
+    </div>
+
+  <div id="footer">
+    <div class="container">
+      <div class='col-xs-12 col-sm-6'>
+        Made proudly by the Methodologists in the U.S.A.
+      </div>
+
+      <div class="col-xs-12 col-sm-6">
+        <%= link_to "About", "#", class: 'pull-right' %>
+        <br />
+        <%= link_to "Contact Us", "#", class: 'pull-right' %>
+        <br />
+        <%= link_to "Register", "#", class: 'pull-right' %>
```

Comment:
@lindyrp probably not especially since register isn't always present in the nav bar. Also, don't think that it needs to replace it. Maybe we can have both?